README.md

```diff
@@ -1,5 +1,117 @@
-#Building Responsive Sites
+# Building Responsive Sites
 
-<iframe width="640" height="480" src="//www.youtube.com/embed/T649edELGoY?rel=0&modestbranding=1" frameborder="0" allowfullscreen></iframe><p><a href="https://www.youtube.com/watch?v=T649edELGoY">Building Responsive Sites</a></p>
+## Problem Statement
+On the modern web, we frequently access the same content on a variety of
+platforms. Each device will display content differently due to their sizes
+and proportions, causing text to shrink, or parts of the page to shift around,
+and can lead to poor user experience. As front end developers, we want our 
+web sites to look good and function well regardless of whether a user is on 
+their laptop, phone or tablet. This has been an ongoing challenge, so what 
+strategies have been developed to address these issues?
+
+## Objectives
+
+1. Using server-side detection for redirection
```

Comment:
Could you make the numbering sequential here and bring the "Explain" verb for points 1-3